tests/test_wave_03.py

```diff
@@ -43,24 +43,28 @@ def test_friends_unique_movies():
     assert FANTASY_4 in friends_unique_movies
     assert amandas_data == clean_wave_3_data()
 
-@pytest.mark.skip()
+# @pytest.mark.skip()
 def test_friends_unique_movies_not_duplicated():
     # Arrange
     amandas_data = clean_wave_3_data()
     amandas_data["friends"][0]["watched"].append(INTRIGUE_3)
+    amandas_data["friends"][0]["watched"].append(HORROR_1)
+    amandas_data["friends"][0]["watched"].append(FANTASY_4)
+    amandas_data["friends"][0]["watched"].append(FANTASY_4)
 
     # Act
     friends_unique_movies = get_friends_unique_watched(amandas_data)
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert INTRIGUE_3 in friends_unique_movies
```

Comment:
I think the instructions here were unclear; the goal was to add the correct movies to the list of asserts without adding lines 51-53:
```suggestion
    assert INTRIGUE_3 in friends_unique_movies
    assert HORROR_1 in friends_unique_movies
    assert FANTASY_4 in friends_unique_movies
```